personalPortfolio/portfolio.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nosipho Manyase</title>
+    <link href = "styles.css" rel="stylesheet" > 
+</head>
+<body>
+  <div class = "first-section">
+  <p class = "my-name">Hi, I'm Nosipho! </p>
+  <img src = "picMe.jpg">
+</div>
+<div class = "second-section">
+  <p class = "my-title">I'm a junior backend developer who loves 
+      solving problems and building solutions.
+  </p>
+  <p class = "intro">Coming from a commerce background with 
+      a BCom degree in Information Systems and Economics, I immediately 
+      delved deep into the realm of Software Engineering and I haven't 
+     looked back since. 
+  </p>
+  <p class = "intro">During my studies, I found it hard to picture 
+      myself working in that field indefinitily. I wanted a career I could picture 
+      myself going to everyday. Once I had made that decision, I had to get out 
+      of my comfort zone and learn a technical skill which I never thought was 
+      in the cards for me.
+  </p>
+  <p class = "intro"> I live by 2 core values (that I may or may not 
+    have made up); knowleadge-seeking and knowleadge-gifting. I aim to 
+    continuously upskill myself and learn more about the field that I 
+    love and I aspire to gift the knowledge that I have amassed to those
+    who are growing up in communities such as I did.
+ </p>
+<h3><span>Technologies I have worked with: </span></h3>
+<ul>
+    <li> Python</li>
+    <li> HTML</li>
+    <li> Java </li>
+    <li> JavaScript </li>
+    <li> CSS </li>
```

Comment:
@NosiphoManyase Once you've done the tutorial that touches on DBs and APIS remember to add that here :)

Reply:
Add git too :)